letsencrypt/interfaces.py

```diff
@@ -199,23 +199,25 @@ class IConfig(zope.interface.Interface):
         "Email used for registration and recovery contact.")
     rsa_key_size = zope.interface.Attribute("Size of the RSA key.")
 
-    config_dir = zope.interface.Attribute("Configuration directory.")
-    work_dir = zope.interface.Attribute("Working directory.")
+    config_dir = zope.interface.Attribute(
+        "Absolute path to the configuration directory.")
+    work_dir = zope.interface.Attribute(
+        "Absolute path to the working directory.")
```

Comment:
Slight problem here. While those descriptions are perfectly OK within this file, they are also used for CLI help generation. In CLI, however, user does not have to use absolute paths. I think that we should stop using `flag_help` in case of all those "absolute path" flags.